html/components/button.stories.js

```diff
@@ -43,39 +41,69 @@ receive interaction.
   },
 };
 
-export const primary = () => (
+export const primary = () =>
   `
     <button class="sprk-c-Button" data-id="button-primary">
       Button
     </button>
-  `
-);
+  `;
 
-export const secondary = () => (
+export const secondary = () =>
   `
-    <button class="sprk-c-Button sprk-c-Button--secondary" type="button" data-id="button-secondary">
+    <button
+      class="sprk-c-Button sprk-c-Button--secondary"
+      type="button"
```

Comment:
I know it was there before and has nothing to do with color, but why do we have type=button as the default here. Sure, it's something that can be added in use cases where its not a submit. But, we don't need to have it as default.